src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -480,7 +481,7 @@
     /**
      * An extends clause.  This appear as part of class and interface
      * definitions.  This element appears even if the
-     * {@code extends} keyword is not explicitly used.  The child
+     * <code>extends</code> keyword is not explicitly used.  The child
```

Comment:
Why do these need to be switched? We just converted these to this type of doc.
See https://github.com/checkstyle/checkstyle/commit/70786709d4f735b9123159a86a752789855af34a#diff-d9a48e7d8e8fa192ed23a1af1597506dR483

Reply:
Because the javadoc utility converts all the inline tags to their HTML counterparts.
If it's ok to have inline tags in the tokentypes.properties file, I'll revert this.

Reply:
I had two options:
* write a converter from inline tags to html tags
* do it manually

"&lt;code&gt;}&lt;/code&gt;" looks better than "{@code &amp;#125;}" in my opinion.

Reply:
done. Just one additional regexp and one line of code.